client/service.go

```diff
@@ -224,11 +225,9 @@ func (srv *Service) subscribe(sub store.Subscription) error {
 		return errors.New("already subscribed to this jobid")
 	}
 
-	var startService func(sub store.Subscription, ch chan subscriber.Event, js *store.JobSpec) (services.Service, error)
-
-	// TODO: Should load entire job spec
+	var startService services.Starter
 	js, err := srv.store.LoadJobSpec(sub.Job)
-	if err != nil || js == nil {
+	if err != nil || gjson.GetBytes(js.Spec, "fluxmonitor").Raw == "null" {
 		startService = srv.subscribeRunlog
```

Comment:
Why not calling `service, err := srv.subscribeRunlog(sub, triggerJobRun, js)` here directly?

Instead of implementing Starter type etc.

Reply:
Good question, I'll change this.